xbout/tests/test_boutdataset.py

```diff
@@ -55,6 +56,35 @@ def test_test_method(self, tmpdir_factory, bout_xyt_example_files):
 
         print(ds.bout.extra_data)
 
+    def test_getFieldAligned(self, tmpdir_factory, bout_xyt_example_files):
+        path = bout_xyt_example_files(tmpdir_factory, nxpe=3, nype=4, nt=1)
+        ds = open_boutdataset(datapath=path, inputfilepath=None, keep_xboundaries=False)
+
+        ds['psixy'] = ds['x']
+        ds['Rxy'] = ds['x']
+        ds['Zxy'] = ds['y']
+
+        ds = apply_geometry(ds, 'toroidal')
+
+        n = ds['n']
+        n.attrs['direction_y'] = 'Standard'
+        n_aligned_from_array = n.bout.toFieldAligned()
+
+        # check n_aligned does not exist yet
+        try:
+            ds['n_aligned']
+            assert False
+        except KeyError:
+            pass
```

Comment:
Is this just `assert 'n_aligned' not in ds`?

Reply:
Yes it is. Sorry I pulled the trigger a bit too quick on this merge. I'll stick the fix in the next PR. Thanks @ZedThree!